test/integration_test.go

```diff
@@ -595,6 +595,47 @@ func TestTerragruntHonorsS3RemoteStateSkipFlagsRegression(t *testing.T) {
 	assert.NotEqual(t, versioningStatus, "Enabled")
 }
 
+// Regression test to ensure that accesslogging_bucket_name is taken into account
+// & sets the TargetLock bucket to a new S3 bucket, different from the origin S3 bucket
+func TestTerragruntSetsAccessLoggingForTfSTateS3BuckeToADifferentBucket(t *testing.T) {
+	t.Parallel()
+	t.Skip("Skipping this test untul PR Terratest-689 has been merged in")
```

Comment:
Looks like that Terratest PR is still in draft state. Is that intentional? Also, you might want to put a TODO above this to make reduce the odds of us forgetting about this!

Reply:
The terratest PR was failing the tests, so wanted to see why they were failing. Will wrap it up as well, once I'm happy the tests are going through